README.md

```diff
@@ -1,4 +1,4 @@
-# tbDEX Protocol
```

Comment:
smushed the relevant bits in the top level readme and the readme in `protocol/`  into one top level readme